sandbox/cgroup/freezer.go

```diff
@@ -49,19 +62,29 @@ func init() {
 // in at most 3000ms. If this time is insufficient then the processes are
 // thawed and an error is returned.
 //
+// A correct implementation is picked depending on cgroup v1 or v2 use in the
+// system. When cgroup v1 is detected, the call will directly act on the device
+// cgroup created when a snap process was started, while with v2 the call will
```

Comment:
confused should be s/device group/freezer group/ ?

Reply:
fixed